content/posts/2015-02-02_february-newsletter.md

```diff
@@ -0,0 +1,130 @@
+will feature some data mining/analysis presentations by Eric Busboom. If
+you're going to the data science workshop, check it out!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
+
+
+[Monthly Meetup][] (meetup.com) <br />
+Our regular monthly meetup will be held at the Ansir Innovation Center on
+Thursday February 26 at 7pm. We are doing 5 minute lightning talks. If you
+are interested in speaking, please [contact us][] or post in the meetup
+comments.
+
+[Monthly Meetup]: http://www.meetup.com/pythonsd/events/220215636/
+[contact us]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+[Data Science Workshop][] (meetup.com) <br />
+San Diego Python in conjunction with the local [R users group][] and the
+[Regional Data Science Library][] is putting on a data science workshop
+at SDSU on February 28. There will be Python and R tracks, data challenges
+and a hackathon.
+
+[Data Science Workshop]: http://www.meetup.com/pythonsd/events/219582270/
+[R users group]: http://www.meetup.com/San-Diego-Data-Science-R-Users-Group/
+[Regional Data Science Library]: http://www.meetup.com/San-Diego-Regional-Data-Library/
+
+
+
+Jobs
+----
+
+[Software developer and craft beer enthusiast][] (workable.com) <br />
+TapHunter, a downtown startup focused on craft beverages, is looking for a
+developer to join their team. Their stack is Python and Django on Appengine
+and their mobile app is written with JavaScript on Titanium.
+
+[Software developer and craft beer enthusiast]: http://taphunter.workable.com/jobs/31725
+
+
+[Software engineer][software-engineer] (jobvite.com) <br />
+Intellisis is looking for you, the agile-loving, curious, technology-obsessed
+Python connoisseur ready to take on new challenges. They're a small company
+near Torrey Pines and are all-in with Python.
+
+[software-engineer]: http://hire.jobvite.com/CompanyJobs/Careers.aspx?c=q8O9VfwK&s=Intellisis&nl=1&page=Job%20Description&j=o7MKZfwm
+
+
+[Software engineer][bs-software-engineer] (brightscope.com) <br />
+BrightScope, a local financial information company in UTC, is looking for
+exceptional engineers to expand their development team. They are big Django
+users and big San Diego Python supporters.
+
+[bs-software-engineer]: http://www.brightscope.com/about/careers/#job_Senior_Software_Engineer
+
+
+News
+----
+
+[January Meetup Video][january-video] (youtube.com) <br />
+Travis Swicegood from [Continuum Analytics][] presented at our
+[January meetup][] on conda, a packaging utility that can be used with their
+popular scientific Python distribution Anaconda. The meetup took place at and
+was sponsored by [NanoImaging Services][].
+
+[january-video]: https://www.youtube.com/watch?v=bJP8NoV7rV4
+[January Meetup]: http://www.meetup.com/pythonsd/events/219205663/
+[Continuum Analytics]: http://continuum.io
+[NanoImaging Services]: http://www.nanoimagingservices.com/
+
+
+[SoCal Linux Expo][socal-linux] (socallinuxexpo.com) <br />
```

Comment:
PyLadies San Diego along with the other SoCal PyLadies groups will be hosting a booth at ....